zeppelin-web/src/index.html

```diff
@@ -45,6 +45,7 @@
     <link rel="stylesheet" href="bower_components/ngtoast/dist/ngToast.css" />
     <!-- endbower -->
     <link rel="stylesheet" href="bower_components/jquery-ui/themes/base/all.css" />
+    <link rel="stylesheet" href="bower_components/ngAnimate/css/ng-animation.css" />
```

Comment:
Could you add license of ng-animation.css in zeppelin-distribution/src/bin_license/LICENSE file? 
For example ngToast, https://github.com/apache/incubator-zeppelin/blob/master/zeppelin-distribution/src/bin_license/LICENSE#L95